tests/integration/integration_test.go

```diff
@@ -79,9 +78,8 @@ func TestEmptyPython(t *testing.T) {
 // TestEmptyGo simply tests that we can build and run an empty Go project.
 func TestEmptyGo(t *testing.T) {
 	integration.ProgramTest(t, &integration.ProgramTestOptions{
-		Dir:      filepath.Join("empty", "go"),
-		Quick:    true,
-		RunBuild: true,
+		Dir:   filepath.Join("empty", "go"),
+		Quick: true,
```

Comment:
Was the change to remove RunBuild required?

Reply:
I am going to retest this one before I merge this

Reply:
yeah this doesn't actually work on Windows for some reason:

```        [                       empty\go ] Updating (p-it-appveyor-v-go-119bcb5f):
         [                       empty\go ] @ Updating....
         [                       empty\go ] 
         [                       empty\go ]     pulumi:pulumi:Stack emptygo-p-it-appveyor-v-go-119bcb5f  error: an unhandled error occurred: problem executing program (could not run language executor): exec: "C:\\Users\\appveyor\\go\\bin\\emptygo": file does not exist
         [                       empty\go ]     pulumi:pulumi:Stack emptygo-p-it-appveyor-v-go-119bcb5f **failed** 1 error
         [                       empty\go ]  
         [                       empty\go ] Diagnostics:
         [                       empty\go ]   pulumi:pulumi:Stack (emptygo-p-it-appveyor-v-go-119bcb5f):
     1>[                       empty\go ]     error : an unhandled error occurred: problem executing program (could not run language executor): exec: "C:\\Users\\appveyor\\go\\bin\\emptygo": file does not exist [C:\Users\appveyor\go\src\github.com\pulumi\pulumi\build.proj]
         [                       empty\go ]  
         [                       empty\go ] Permalink: https://app.pulumi-staging.io/pulumi-bot/emptygo/p-it-appveyor-v-go-119bcb5f/updates/1
         [                       empty\go ] 
         [                       empty\go ] Wrote output to empty\go\command-output\pulumi-update-initial.20200127-170806.53f1c.log
         [                       empty\go ] Destroying stack
         [                       empty\go ] **** Invoke 'C:\Pulumi\bin\pulumi.exe destroy --non-interactive --skip-preview' in 'empty\go'
         [                       empty\go ] Wrote output to empty\go\command-output\pulumi-destroy.20200127-170809.eeda1.log
         [                       empty\go ] **** Invoke 'C:\Pulumi\bin\pulumi.exe stack rm --yes' in 'empty\go'
         [                single_resource ] Wrote output to C:\Users\appveyor\AppData\Local\Temp\1\p-it-appveyor-v-single_res-95c9f6be-584558113\command-output\pulumi-update-initial.20200127-170810.855a7.log
```

Thoughts on this?

Reply:
FWIW, this is the only empty go test that has this...